.github/workflows/forge-build-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Forge Build Size Check
+
+on:
+  push:
+    paths:
+      - 'packages/contracts-core/**'
+      - '.github/workflows/foundry-build-check.yml'
+  pull_request:
+    paths:
+      - 'packages/contracts-core/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Setup Node JS
+        uses: ./.github/actions/setup-nodejs
+
+      - name: Installing dependencies
```

Comment:
Not needed, setup-nodejs will take care of this

Reply:
see CI committed by @trajan0x for which this action was based on, is this one wrong as well?

https://github.com/synapsecns/sanguine/blob/master/.github/workflows/foundry-tests.yml#L34

Reply:
yessir